src/utils/jsonable.js

```diff
@@ -0,0 +1,39 @@
+/* @flow strict */
+
+/**
+ * Type of a round-trip-capable JSONable object.
+ *
+ * Note that this is not the same as the type of objects which `JSON.stringify`
+ * will accept, nor even the type of objects which `JSON.stringify` will accept
+ * without returning `undefined`. For example,
+ *
+ *    JSON.stringify({f: 2, g: () => 3, h: undefined}) === '{"f":2}'
+ *
+ * which of course will not be equal to the original object after parsing.
+ */
+// eslint-disable-next-line flowtype/type-id-match
+export type JSONable =
+  | null
+  | string
+  | number
+  | boolean
+  | { +[string]: JSONable } /* [α] */
+  | $ReadOnlyArray<JSONable>;
+
+// [α]: This should really be an exact type, `{| +[string]: JSONable |}`.
+// Unfortunately, it can't yet be.
+//
+// Prior to Flow v0.111.0 (i.e., prior to React Native v0.62.0), exact object
+// types with indexer properties are unusably broken. The following trivial
```

Comment:
That's good news that there *is* now a version of Flow where indexer properties on exact object types work reasonably! Please mention that on the RN v0.62 upgrade issue, too :slightly_smiling_face: